paddle/fluid/framework/tensor.h

```diff
@@ -37,23 +37,84 @@ class Tensor {
 #ifdef PADDLE_WITH_MKLDNN
 
  public:
-  inline mkldnn::memory::format format() const { return format_; }
+  // TODO(jczaja): This is depracted and will be removed
+  inline mkldnn::memory::format format() const {
```

Comment:
what if these format are put in Paddle's Layout enum? will it be cleaner without these ifdef PADDLE_WITH_MKLDNN